docs/common/top-bar.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, SvgIcon, Typography } from '@mui/material';
+import * as React from 'react';
+
+interface Props {
+  icon?: React.ElementType;
+  title: JSX.Element | string;
+}
+
+const TopBar: React.FC<Props> = ({ icon, title }) => {
+  return (
+    <Box
+      sx={{
+        height: '68px',
+        bgcolor: '#555555',
+        display: 'flex',
+        alignItems: 'center',
+        padding: '0px 1rem',
+        justifyContent: 'flex-start',
+      }}
+    >
+      {icon && (
+        <SvgIcon

```

Comment:
Is having `SvgIcon` here redundant?  I think you can directly display the Icon, but correct me if I am wrong / check with Ian.

Reply:
Interesting, yeah I think that as is `icon` is a `ElementType` rather than an element, and you can't render the element from element type once its in the variable.

If you did pass the icon in as `icon={<SmallLightLogo/>}`, you have more luck rendering it later as `{icon}` but then styles need to be passed from the parent... so also not ideal, you can leave it as is.